src/registrar/management/commands/master_domain_migrations.py

```diff
@@ -0,0 +1,540 @@
+    Domain,
+    DomainInformation,
+    DomainInvitation,
+    TransitionDomain,
+)
+
+from registrar.management.commands.utility.terminal_helper import (
+    TerminalColors,
+    TerminalHelper,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = """ """
+
+    def add_arguments(self, parser):
+        """
+        OPTIONAL ARGUMENTS:
+        --runMigrations
+        A boolean (default to true), which triggers running
+        all scripts (in sequence) for transition domain migrations
+
+        --migrationDirectory
+        The location of the files used for load_transition_domain migration script
+        EXAMPLE USAGE:
+        > --migrationDirectory /app/tmp
+
+        --migrationFilenames
+        The files used for load_transition_domain migration script.
+        Must appear IN ORDER and comma-delimiteds:
+        EXAMPLE USAGE:
+        > --migrationFilenames domain_contacts_filename.txt,contacts_filename.txt,domain_statuses_filename.txt
+        where...
+        - domain_contacts_filename is the Data file with domain contact information
+        - contacts_filename is the Data file with contact information
+        - domain_statuses_filename is the Data file with domain status information
+
+        --sep
+        Delimiter for the migration scripts to correctly parse the given text files.
+        (usually this can remain at default value of |)
+
+        --debug
+        A boolean (default to true), which activates additional print statements
+
+        --prompt
+        A boolean (default to true), which activates terminal prompts
+        that allows the user to step through each portion of this
+        script.
+
+        --limitParse
+        Used by the migration scripts (load_transition_domain) to set the limit for the
+        number of data entries to insert.  Set to 0 (or just don't use this
+        argument) to parse every entry. This was provided primarily for testing
+        purposes
+
+        --resetTable
+        Used by the migration scripts to trigger a prompt for deleting all table entries.
+        Useful for testing purposes, but USE WITH CAUTION
+        """  # noqa - line length, impacts readability
+
+        parser.add_argument(
+            "--runMigrations",
+            help="Runs all scripts (in sequence) for transition domain migrations",
+            action=argparse.BooleanOptionalAction,
+        )
+
+        # --triggerLogins
+        # A boolean (default to true), which triggers running
+        # simulations of user logins for each user in domain invitation
+        parser.add_argument(
+            "--triggerLogins",
+            help="Simulates a user login for each user in domain invitation",
+            action=argparse.BooleanOptionalAction,
+        )
+
+        # The following file arguments have default values for running in the sandbox
+        parser.add_argument(
+            "--migrationDirectory",
+            default="migrationData",
```

Comment:
**question (non-blocking)**: as above, shouldn't this be `migrationdata`?

Reply:
@zandercymatics - can you resolve?